src/index.js

```diff
@@ -475,62 +495,250 @@ const adaptive_test_node = {
 };
 
 // trials to add gamification
-const preload_intro_video = {
+const images = [
+  left_arrow_image,
+  right_arrow_image,
+];
+
+const videos = [
+  intro_video_1,
+  intro_video_2,
+  intro_video_3,
+  game_instructions_1,
```

Comment:
It's generally helpful to have a prefix or a suffix describing what the variable holds. Something like `game_instructions_1_video` or `video_game_instructions_1`.
If you can change this for all the videos by using the suffix for example, that would be great.